your-rents-services-geodata/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1 @@
+<configuration/>
```

Comment:
Why this file? You can configure the loggers in `application-test.properties`.

Reply:
we used it also in your-rents backend: https://github.com/your-rents/your-rents-backend/blob/develop/your-rents-api/src/test/resources/logback-test.xml
I would prefer to keep the log configuration separated from the application properties